vendor/k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/stats/types.go

```diff
@@ -168,7 +168,7 @@ type MemoryStats struct {
 	// dirty memory, and kernel memory. WorkingSetBytes is <= UsageBytes
 	// +optional
 	WorkingSetBytes *uint64 `json:"workingSetBytes,omitempty"`
-	// The amount of anonymous and swap cache memory (includes transparent
+	// The amount of anonymous memory (includes transparent
```

Comment:
> Am I allowed to sign CLA ?

Yes, everyone can and should sign the CLA. :)

> Do you know who can verify the meaning of rss here ?

The way you explain about `rss` is right. `rss` contains memory excluding cached ones. We can understand `usage` ~= `rss` + `cached`.